src/components/ui/RotationTable.tsx

```diff
@@ -83,7 +83,7 @@ export interface RotationTableEntry {
 	/**
 	 * Rotation to display that occurs during this entry
 	 */
-	rotation: Array<{ability: Ability}>
```

Comment:
mmm. this'll probably need some thinkin' 'bout for analyser events, but it'll work nice for now